sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/AddUserToOrgTest.java

```diff
@@ -50,4 +51,34 @@ public void testAddUserToOrgFailure(String testName) {
         HttpStatus.BAD_REQUEST,
         RESPONSE_JSON);
   }
+
+  @Test()
```

Comment:
Add this test within success data provider.

Reply:
changes has been incorporated